### PR TITLE
Massmint - Table + Edit Panel + Dialogs

### LINKED ISSUE #5180: Mass Mint: step 4 - Overview table
* #4964 

## Figma
---
- [Detailed interactions and logic is available Figma](https://www.figma.com/file/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?node-id=0%3A1&t=ynDINS3WQGGG9Dj0-1). 
- all the needed screens
- [interactive prototype ](https://www.figma.com/proto/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?page-id=0%3A1&node-id=1%3A332&viewport=1014%2C587%2C0.04&scaling=scale-down-width&starting-point-node-id=1%3A332&show-proto-sidebar=1)

Please take the prototype with a grain of salt, not everything is perfect and 1:1 to final design how it should look, so in case you don't know something, don't hesitate to ask @VladoTheBoi 

## STEP 4: OVERVIEW TABLE
image
---

Mint Button:
- by opening this table Mint button becomes tappable
  - mint button shows how many NFTs will be minted

Table:
- table shows uploaded art files 
- each file is displayed as a separate row
- row with columns description is fixed and does not move  
- user can scroll in the frame with NFTs starting with#1
- would be nice if the height of the table is resizable (nice to have)

List Item in Table:
- **Name** 
  - ~~taken from the name of the uploaded files~~
  - in case user uploads description file, name and other information is taken from that file 
  - if missing, “*Name Required” is displayed
  - whole name textfield can also be tapped to open edit panel

- **Description**
  - taken from description file 
  - if empty, display “Description missing”
  - whole description textfield can also be tapped to open edit panel

- **Price** 
  - taken from description file 
  - if empty, display “*Price Required”
  - whole price textfield can also be tapped to open edit panel

- **States**
  - **OK** - this is desired state so that the file can be minted
    - Name, Price and Description are filled
  - **Incomplete**
    - file is missing required information:
      - Name
      - Price
      - or Both
    - If some files in the table have this state, user cannot mint unless he resolves them. If he tries to mint “Incomplete mint - popup” is displayed 
  - **Description** 
    - file which is missing only description
    - this state does not restrict a user from minting 
- **Operation** 
  - Edit
    - users can edit each file by clicking edit button 
this opens a side edit panel where all changes to the image can be done
  - Delete
    - users can also delete an item from the table
    - confirmation popup is displayed after delete button is tapped

### LINKED ISSUE #5181: Mass Mint: Edit Panel
* #4964 

## Figma
---
- [Detailed interactions and logic is available Figma](https://www.figma.com/file/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?node-id=0%3A1&t=ynDINS3WQGGG9Dj0-1). 
- all the needed screens
- [interactive prototype ](https://www.figma.com/proto/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?page-id=0%3A1&node-id=1%3A332&viewport=1014%2C587%2C0.04&scaling=scale-down-width&starting-point-node-id=1%3A332&show-proto-sidebar=1)

Please take the prototype with a grain of salt, not everything is perfect and 1:1 to final design how it should look, so in case you don't know something, don't hesitate to ask @VladoTheBoi 

## Edit Panel

- Clicking on the edit button in the table will open the edit panel from the right, check prototype
- when missing required information (Name, price) textfields are red and placeholder says “*Required”
- description is limited at 500 characters
- save button can be tapped when changes were made and no *Required information is missing 
- user can always go back by tapping close button without any changes
---
image
---
image

### LINKED ISSUE #5182: Mass Mint: Dialogs
* #4964 

## Figma
---
- [Detailed interactions and logic is available Figma](https://www.figma.com/file/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?node-id=0%3A1&t=ynDINS3WQGGG9Dj0-1). 
- all the needed screens
- [interactive prototype ](https://www.figma.com/proto/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?page-id=0%3A1&node-id=1%3A332&viewport=1014%2C587%2C0.04&scaling=scale-down-width&starting-point-node-id=1%3A332&show-proto-sidebar=1)

Please take the prototype with a grain of salt, not everything is perfect and 1:1 to final design how it should look, so in case you don't know something, don't hesitate to ask @VladoTheBoi 

## Dialogs
There are 4 main dialogs:
1. **Missing Required Information** 
  - this dialog is shown when there are some files with Incomplete state
  - user cannot mint unless he resolves the incomplete files
  - can be resolved by:
    - filling in required information
    - deleting incomplete files 

image
---

2. **Final Mass Mint Overview - missing description**
  - when Description missing, it does not restrict user from minting 
  - he is however informed that some files are missing description 
  - shows how many NFTs will be minted
  
image
---

3 **Final Mass Mint Overview - Everything OK**
  - shows how many NFTs will be minted
  image
image
image

---
4. **Delete file** 
  - displayed when user taps delete file button 
  - if name is missing, display file number from the table
image

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5180 
- [x] Closes #5181 
- [x] Closes #5182 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Testing

- [x]  https://deploy-preview-5765--koda-canary.netlify.app/bsx/massmint

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 77d56d2</samp>

This pull request implements the mass mint feature for the NFT gallery, which allows users to upload a zip file of images and mint multiple NFTs with metadata. It adds and modifies several components, such as `Massmint`, `OverviewTable`, `EditPanel`, and various modals, to handle the user interface and logic of the feature. It also adds some types, utility classes, and props to the `NeoModal` and `NeoSidebar` components to support the feature.

### <samp>🤖 Generated by Copilot at 77d56d2</samp>

> _We are the masters of the mint_
> _We create and destroy with a click_
> _We unleash the power of the `NFT`_
> _We rule the gallery with the `NeoModal` and the `NeoSidebar`_

### CODE REVIEW

**Comment by daiagi** [comment]
### It doesn't actually mint yest

**Comment by exezbcz** [comment]
hey hey, nice job! Mass mint almost done! 

## Feedback 
- it will be bit harder for my since I did not fully make the designs, but I will do my best to identify the issues 

- I would maybe remove the footer completely on mass mint and these subpages
- should scroll be enabled when edit sidebar is opened? 
image
- can you please add some padding into the text container
image


image
- this text to normal weight 

please see this screen: 
image
- the column names should be grey, operation icons as well 
- divider to grey please as well 
image



image
- picture preview should work the same as on gallery cards - that means the img is scaled, not stretched 

description input limit missing
image


the sidebar is too wide, can you make it smaller please 


image
- the input fields should have more height - similar for example to the buy button in gallery item detail: 
image


the tags are bit different from the design
image
- we can maybe use the same that are in collection activity - with a change of the pink to more red-ish color (#FFB6B6) 
- they have also dark mode alternative
image
image

I may notice something else, but these are some starting points:D

**Comment by daiagi** [comment]
@exezbcz  all done

**Comment by exezbcz** [comment]
letsgo! next batch :D 


### Next
image
- less blur please, match it to the wallet/notification sidebar 

maybe show the word count all the time? 
image


can we make the buttons same width please? 
image


I have issue with loading some previews: 
image


It does not show the correct number of imgs, can you check that as well please
image
- I uploaded only 3 imgs, not 5, 
- 3 of them are not loading

**Comment by daiagi** [comment]
@exezbcz  All Done

**Comment by exezbcz** [comment]
@daiagi 

I will check again soon

**Comment by exezbcz** [comment]
woho, everything from the visual side is fine! 
Solving osx zip folder bug be like: 
image

File: components/massmint/OverviewTable.vue [hunk 1]
```diff
@@ -5,21 +5,191 @@
...
+            <div class="column is-flex is-align-items-center">
+              <img :src="nft.imageUrl" class="image is-48x48 border cover" />
+            </div>
+            <div class="column is-flex is-align-items-center">
+              <div
+                class="is-clickable"
+                :class="{
+                  'has-text-k-red': !nft.name,
+                  'has-text-grey': nft.name,
+                }"
```
Comment (floyd-li) [inline components/massmint/OverviewTable.vue:38]: is it better to use conditional operator like `nft.name ? 'has-text-grey' : 'has-text-k-red'`?

File: components/massmint/OverviewTable.vue [hunk 2]
```diff
@@ -5,21 +5,191 @@
...
+                {{ nft.name || '*' + $t('massmint.nameRequired') }}
+              </div>
+            </div>
+            <div class="column is-3 is-flex is-align-items-center">
+              <div
+                class="is-clickable"
+                :class="{
+                  'has-text-k-red': !nft.description,
+                  'clip-text has-text-grey': nft.description,
+                }"
```
Comment (floyd-li) [inline components/massmint/OverviewTable.vue:49]: same, the conditional operator

File: components/massmint/OverviewTable.vue (hunk 1 above)
Comment (daiagi) [reply components/massmint/OverviewTable.vue:38]: that would require creating a new  computed ref, which i didn't want to do.
https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes

File: components/massmint/EditPanel.vue [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        class="is-flex w-full is-flex-direction-column is-justify-content-space-between is-align-items-center">
+        <div class="is-flex w-full">
+          <div class="is-flex is-justify-content-center is-flex-grow-1">
+            {{ $t('massmint.edit') }} #{{ nft?.id }}
+          </div>
+          <NeoIcon
+            icon="close"
+            size="small"
+            class="is-clickable"
+            @click.native="closePanel" />
```
Comment (roiLeo) [inline components/massmint/EditPanel.vue:23]: click on icon?

File: components/massmint/EditPanel.vue [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        </div>
+        <img
+          :src="nft?.imageUrl"
+          class="image is-128x128 cover border k-shadow my-5" />
+        <div class="form w-full">
+          <o-field
+            :label="$t('massmint.name')"
+            class="w-full mb-4 placholder-color"
+            :class="{ 'red-border': !name }">
+            <o-input
```
Comment (roiLeo) [inline components/massmint/EditPanel.vue:33]: 😢 `NeoInput`?

File: components/massmint/EditPanel.vue [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+              :placeholder="'*' + $t('massmint.required')"
+              class="field-height" />
+          </o-field>
+          <o-field :label="$t('massmint.description')" class="w-full mb-4">
+            <o-input
+              v-model="description"
+              type="textarea"
+              has-counter
+              maxlength="500" />
+            <!-- <div>{{ description.length }}/500</div> -->
```
Comment (roiLeo) [inline components/massmint/EditPanel.vue:44]:
```suggestion
```

File: components/massmint/EditPanel.vue [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+      border: 1px solid theme('text-color');
+      background-color: theme('background-color');
+      color: theme('text-color');
+    }
+    padding: 0.5rem;
+  }
+}
+
+.o-field--focused:deep {
+  input,
```
Comment (roiLeo) [inline components/massmint/EditPanel.vue:180]: so many custom css lines

File: components/massmint/OverviewTable.vue [hunk 7]
```diff
@@ -5,21 +5,191 @@
...
+  return status ? statusMap[status] : ''
+}
+
+const addStatus = (nft: NFT): NFT => {
+  let status: Status = 'Ok'
+  if (!nft.description) {
+    status = 'Description'
+  }
+  if (!nft.name) {
+    status = 'Incomplete'
```
Comment (roiLeo) [inline components/massmint/OverviewTable.vue:140]: i18n for status?

File: components/massmint/OverviewTable.vue [hunk 8]
```diff
@@ -5,21 +5,191 @@
...
+.height-50px {
+  height: 50px;
+}
+
+//colums overrides
+.column {
+  padding: 0;
+  padding-right: 0.25rem;
+}
+
```
Comment (roiLeo) [inline components/massmint/OverviewTable.vue:188]: this isn't right, maybe check [column gap](https://bulma.io/documentation/columns/gap)

File: components/massmint/modals/MintingModal.vue [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  loading: boolean
+}>()
+
+const isModalActive = useVModel(props, 'value')
+
+const emit = defineEmits(['close'])
+</script>
+
+<style lang="scss" scoped>
+.check-icon {
```
Comment (roiLeo) [inline components/massmint/modals/MintingModal.vue:45]: maybe check `NeoIcon` size prop (`large`)

File: components/massmint/modals/ReviewModal.vue [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+const isModalActive = useVModel(props, 'value')
+
+const emit = defineEmits(['close', 'mint'])
+</script>
+
+<style lang="scss" scoped>
+li {
+  list-style-type: disc;
+}
```
Comment (roiLeo) [inline components/massmint/modals/ReviewModal.vue:74]: do you really need to use list only for one element?

File: components/massmint/OverviewTable.vue (hunk 1 above)
Comment (floyd-li) [reply components/massmint/OverviewTable.vue:38]: > that would require creating a new computed ref

i'm not clear, why? there's an example in the doc you provided
> If you would like to also toggle a class in the list conditionally, you can do it with a ternary expression:
`<div :class="[isActive ? activeClass : '', errorClass]"></div>`

but nvm, it's just a small suggestion, you can do it your way :)

File: components/massmint/modals/MintingModal.vue (hunk 9 above)
Comment (daiagi) [reply components/massmint/modals/MintingModal.vue:45]: that prop doesn't work

File: components/massmint/EditPanel.vue (hunk 3 above)
Comment (daiagi) [reply components/massmint/EditPanel.vue:23]: can change to NeoButton i guess

File: components/massmint/EditPanel.vue (hunk 6 above)
Comment (daiagi) [reply components/massmint/EditPanel.vue:179]: yeah well... what would you suggest?
needed some styling, and I'm not sure it justifies creating a NeoInput component

Comment (roiLeo) [reply components/massmint/EditPanel.vue:179]: > yeah well... what would you suggest? needed some styling, and I'm not sure it justifies creating a NeoInput component

Create `NeoInput` then add [custom css](https://oruga.io/components/Input.html#style)
I'm a bit tired of seeing oruga be used where it should not, I'm pretty sure we'll need this component afterward

File: components/massmint/EditPanel.vue (hunk 3 above)
Comment (roiLeo) [reply components/massmint/EditPanel.vue:23]: > can change to NeoButton i guess

*Should change, better accessibility and no side effect stuff

File: components/massmint/modals/MintingModal.vue (hunk 9 above)
Comment (roiLeo) [reply components/massmint/modals/MintingModal.vue:45]: Aight, will open an issue

**Comment by roiLeo** [comment]
> ## Feedback

> 
> image

Hey @exezbcz this modal looks weird, wouldn't it better to have CTA on "Yes DELETE" and "Cancel" to secondary action?
(change orders of buttons)

File: components/massmint/OverviewTable.vue (hunk 8 above)
Comment (daiagi) [reply components/massmint/OverviewTable.vue:188]: ```is-gapless``` works
but is-<some-number> doesn't work (tried on chrome and firefox)

so i'd still need custom css

Comment (roiLeo) [reply components/massmint/OverviewTable.vue:188]: Have you tried something like this `columns is-variable is-2`?

File: components/massmint/EditPanel.vue (hunk 3 above)
Comment (daiagi) [reply components/massmint/EditPanel.vue:23]: aight

mind you, it is NeoIcon in connectWallet and notifications modals as well

Comment (roiLeo) [reply components/massmint/EditPanel.vue:23]: > mind you, it is NeoIcon in connectWallet and notifications modals as well

Sorry I can't be on all reviews 😭. if you can't, it's okay, we'll see later

**Comment by VladoTheBoi** [comment]
@daiagi do you still need edit access to the Massmint Figma file?

**Comment by VladoTheBoi** [comment]
> Hey @exezbcz this modal looks weird, wouldn't it better to have CTA on "Yes DELETE" and "Cancel" to secondary action?
(change orders of buttons)

Yes, we can change the colors.

**Comment by daiagi** [comment]
> do you still need edit access to the Massmint Figma file? 
@VladoTheBoi  no. thanks

File: components/massmint/OverviewTable.vue (hunk 8 above)
Comment (daiagi) [reply components/massmint/OverviewTable.vue:188]: you are right. Thank you!!

Didn't read the docs carefully enough :disappointed:

**Comment by VladoTheBoi** [comment]
>Hey @exezbcz this modal looks weird, wouldn't it better to have CTA on "Yes DELETE" and "Cancel" to secondary action?
(change orders of buttons)

Checked the dialog card and have some comments if possible:
- remove the shadow from the buttons within the dialog (looked at it after long time and double shadow does not work together)
- set padding to 32px all around the dialog
- set padding to 32px between sections and 16px within section
- set the default size of the modal to 416px (so that all the dialogs are same width) 
- set the height of the buttons to 54px (same size as Buy or Make Offer buttons)
- Title of the dialog is "Text-semilarge" = 20px (also not sure how the line height of the text is set right now?)
  - Bold Title is used only in dialog with required information are missing
  - Delete dialog - make bold the name of the item, rest not bold (changed in designs now) + add separator under the title to be consistent (also added now)


Screenshot 2023-04-25 at 11 55 05

Delete confirmation

File: components/massmint/EditPanel.vue (hunk 6 above)
Comment (roiLeo) [reply components/massmint/EditPanel.vue]: in libs/ui:
- change props `modelValue` to `value`
- remove vueuse import?
```ts const emit = defineEmits(['input'])
const inputValue = useVModel(props, 'value')
const updateValue = (event) => {
  emit('input', event)
}
```
&
```vue
<template>
  <o-input
    v-model="inputValue"
    class="neo-input"
    v-bind="$attrs"
    :style="heightStyle"
    @input="updateValue" />
</template>
```

This work for me

File: components/massmint/modals/DeleteModal.vue [hunk 11]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <NeoModal v-model="isModalActive" scroll="clip" @close="emit('close')">
+    <div class="p-7">
+      <header class="">
```
Comment (roiLeo) [inline components/massmint/modals/DeleteModal.vue:4]:
```suggestion
      <header>
```

**Review by roiLeo — COMMENTED** [review]
you're missing:
- oruga ui input style import (checkout neocheckbox scss file)
- required `NeoInput` prop 
- placeholder `NeoInput` prop (!== `label`)
- use form in `EditPanel` to trigger html validation
- clickable edit/delete icon???
- https://github.com/kodadot/nft-gallery/pull/5765#discussion_r1177579196

Translation file is not very optimal but this can go next

File: components/massmint/EditPanel.vue (hunk 6 above)
Comment (daiagi) [reply components/massmint/EditPanel.vue]: Done

:clinking_glasses:  Thank you

**Comment by daiagi** [comment]
> Checked the dialog card and have some comments if possible:


@VladoTheBoi  Implemented

**Comment by VladoTheBoi** [comment]
> > Checked the dialog card and have some comments if possible:
> 
> @VladoTheBoi Implemented

Could you please post a screenshots of some of those dialogs please? 
I am having trouble to get some tokens on BSX so that I could create a collection and test it myself.

**Comment by daiagi** [comment]
> you're missing:
> 
>     * oruga ui input style import (checkout neocheckbox scss file)
> 
>     * placeholder `NeoInput` prop (!== `label`)
> 
>     * use form in `EditPanel` to trigger html validation
> 
>     * clickable edit/delete icon???
> 
>     * [Massmint - Table + Edit Panel + Dialogs #5765 (comment)](https://github.com/kodadot/nft-gallery/pull/5765#discussion_r1177579196)

Done

> * required `NeoInput` prop

what you mean?

**Comment by roiLeo** [comment]
> > * required `NeoInput` prop
> 
> what you mean?

html `required` attr for form validation

**Comment by daiagi** [comment]
@VladoTheBoi  added pics in the top

**Comment by daiagi** [comment]
> html `required` attr for form validation
Done

**Review by roiLeo — APPROVED** [review]
✅ wfm

slight subsequent change:
- input label dark mode color in editPanel
- .dark-mode .k-yellow?
- price missing? not needed. if I want to mint without setting price
- same for description maybe change color (warn: orange or info: blue)
- delete modal change button order (pink CTA should be yes delete & cancel should be secondary)
- different txt color between table content & column header
- align buttons to right in table

**Comment by VladoTheBoi** [comment]
> @VladoTheBoi added pics in the top

@daiagi yes, awesome, the dialogs look great! Thank you.

I will take take a look at the rest of the feature as well and will create separate tickets for some UI inconsistencies.

**Comment by daiagi** [comment]
>     * input label dark mode color in editPanel
yep .missed that


---


>  * .dark-mode .k-yellow?

 yep, that's the color
image

---
>     * price missing? not needed. if I want to mint without setting price
> 
>     * same for description maybe change color (warn: orange or info: blue)

agree.
@VladoTheBoi  let's pick a new color for this please

---

>     * delete modal change button order (pink CTA should be yes delete & cancel should be secondary)

sure thing

---

>     * different txt color between table content & column header

@VladoTheBoi  header to black? values to black? or something else?


---

>     * align buttons to right in table

Sure thing.

**Review by Jarsen136 — APPROVED** [review]

**Comment by VladoTheBoi** [comment]
> @VladoTheBoi header to black? values to black? or something else?
- lets make the content Black color, and put a k-grey (#999999) bottom stroke on the header

> agree. @VladoTheBoi let's pick a new color for this please
The reason why I left the red for description and price was because of contrast as the orange/yellow is not really visible on white bg. I will play around and try to find a way how to distinguish the color, or some other way. For now try to use this dark yellow #CF9A10. 

also some minor things we can solve as well:
- change status tag padding to 8px, 16px
- align operation icons to center of the operations header
- make operation icons 24x24px
- align status tag to the center of the status header as well
- move image + name and description columns more to the left, as we dont need the empty space next to the number ID. (will add more comments in separate ticket)
Screenshot 2023-04-27 at 19 56 22

- since price is no more required, make new yellow status "optional" for case when description + price is missing (if only price is missing, show yellow "price" status, if description is missing show yellow "description" status)
  -  add "Missing price: X NFTs" for "final overview" and "required information missing" dialogs
  - add colon ":" accordingly in dialogs see screenshot below
Screenshot 2023-04-27 at 20 27 03
Screenshot 2023-04-27 at 20 28 52


- change the number in the Mint Button based on how many NFTs can be minted at that moment (images that are not incomplete)
  - make the number and brackets bold and move to the end of the button 
  - make button bigger to 59px 
  - use bigger 20px font in the button
Screenshot 2023-04-27 at 20 34 06

If its exhaustive, we can move it to separate ticket.

**Comment by daiagi** [comment]
aight @VladoTheBoi 

all done.
let's let it roll :roller_skate:

**Comment by daiagi** [comment]
btw, not sure what's up with DeepSource
all of these issues are wrong (i.e not true, the vars are used)

image

**Comment by VladoTheBoi** [comment]
> aight @VladoTheBoi
> 
> all done. let's let it roll 🛼

Great, thank you! 
Will gather next batch of comments in separate issue.

**Comment by daiagi** [comment]
@vikiival @yangwao  merge?

**Review by preschian — APPROVED** [review]

**Comment by preschian** [comment]
> btw, not sure what's up with DeepSource
> all of these issues are wrong (i.e not true, the vars are used)

seems like DeepSource is unable to scan the `.vue` component properly

**Comment by daiagi** [comment]
@preschian do you have any idea what to do to solve it?

**Comment by preschian** [comment]
> @preschian do you have any idea what to do to solve it?

no idea, I think it is safe to ignore for that case

**Comment by daiagi** [comment]
image

**Comment by daiagi** [comment]
@vikiival  why no merge?

**Comment by vikiival** [comment]
Pinging @yangwao about payout

**Comment by yangwao** [comment]
pay 200 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $200 @ 24.95 USD/KSM ~ 8.016 $KSM
🧗 [EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L](https://kusama.subscan.io/account/EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)
🔗 [0xeb2ecffae95774e71deb9d8a36aeeea2b6075d2e0c388568b566be24eb321b72](https://kusama.subscan.io/extrinsic/0xeb2ecffae95774e71deb9d8a36aeeea2b6075d2e0c388568b566be24eb321b72)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**